app/views/top_tens/index.html.erb

```diff
@@ -0,0 +1,81 @@
+<!-- top work -->
+<section class="media-spotlight">
+
+  <h3><span class="media-spotlight-header">Media Spotlight: </span>
+  <% top_work = Work.top_work %>
+  <%= link_to top_work.title, work_path(top_work) %>
+  by <%= top_work.creator %>
+  </h3>
+  <p>
+    <%= top_work.votes.count %> votes - <%= top_work.description %>
+  </p>
+
+</section>
+
+<hr />
+
+<!-- top tens -->
+
+<section class="top-ten-container">
```

Comment:
I'd love to see you try to move each of these sections into a partial that is customized for each type of work